android/app/src/main/java/co/ello/ElloApp/MainActivity.java

```diff
@@ -338,13 +338,13 @@ private void setupRegisterDeviceReceiver() {
             @Override
             public void onReceive(Context context, Intent intent) {
                 String reg_id = intent.getExtras().getString("GCM_REG_ID");
-                String registerFunctionCall =
+                if(reg_id != null) {
+                    String registerFunctionCall =
                         "javascript:registerAndroidNotifications(\"" +
                                 reg_id + "\", \"" +
                                 packageName() + "\", \"" +
                                 versionName() + "\", \"" +
                                 versionCode() + "\")";
-                if(reg_id != null) {
```

Comment:
Good catch, @jayzes. Looks like I was too hasty in getting this up. Changes incoming.